architecture.md

```diff
@@ -544,7 +544,7 @@ dedicated master node) calls into a compute kubelet. This kubelet instance is
 responsible for managing the lifecycle of pods within the nodes and eventually relies
 on a container runtime to handle execution. The kubelet architecture decouples
 lifecycle management from container execution through the dedicated
-[`gRPC`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto)
+[`gRPC`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto)
```

Comment:
`v1alpha2` - I had to go check, coz that was not inspiring me with confidence :-)

Reply:
I totally agree, but it looks like that's as good as it gets...